test/test_qml/test_messagel_mode.h

```diff
@@ -1,5 +1,6 @@
 #ifndef TESTCASE_H
 #define TESTCASE_H
```

Comment:
```suggestion
#define TESTMESSAGEMODEL_H
```
может переименовать класс